browser/src/control/Control.Menubar.js

```diff
@@ -1411,7 +1412,7 @@ L.Control.Menubar = L.Control.extend({
 	},
 
 	_beforeShow: function(e, menu) {
-		$('.main-nav.hasnotebookbar').css('overflow', 'visible');
+		$('.main-nav').css('overflow', 'visible');
```

Comment:
When switching classic -> nb -> classig I've got:
![switch](https://user-images.githubusercontent.com/5307369/155132436-54e4f11e-b19e-4c59-a213-c47959c8ebe1.png)

this line can be a cause

Reply:
I can still reproduce this issue in Chrome:
https://user-images.githubusercontent.com/5307369/159482695-f349ffc1-3372-4d45-be05-4420ca936198.mp4